Sources/UIViewExtensions.swift

```diff
@@ -307,6 +309,28 @@ extension UIView {
         self.layer.transform = transform
     }
 
+    
+    /// EZSwiftExtensions
+    public func setTranslation(_ duration:TimeInterval?, _ x: CGFloat?, y: CGFloat?, completion: ((Bool) -> Void)? = nil) {
+        self.animate(duration: duration ?? UIViewTransitionDuration, animations: {
+            self.transform = .init(translationX: x ?? 0, y: y ?? 0)
+        }) { (success) in
+            if let completion = completion {
+                completion(success)
+            }
+        }
+    }
+    
+    /// EZSwiftExtensions
+    public func setTranslationIdentity(_ duration:TimeInterval?, completion: ((Bool) -> Void)? = nil) {
+        self.animate(duration: duration ?? UIViewTransitionDuration, animations: {
+            self.transform = CGAffineTransform.identity
```

Comment:
this function is only to back to his origin after translation, rotate, scale, etc..  I'll change the name.